lib/pages/home_page.dart

```diff
@@ -27,6 +29,8 @@ class HomePage extends HookWidget {
     final theme = Theme.of(context);
     final currentTab = useState(0);
     final snackBarShowing = useState(false);
+    final accStore = useAccountsStore();
+    final inboxStore = useStore((InboxStore store) => store);
```

Comment:
Seems unnecessary to use useStore here with a selector that just selects the store. Probably no big deal but I think it'd make more sense to replicate the useAccountsStore() function in the stores.dart file with a new function called useInboxStore().